server/app.ts

```diff
@@ -190,20 +85,18 @@ let root = {
 apiRouter.use("/user", userRoutes);
 
 app.use("/api", apiRouter);
-app.use('/graphql', express_graphql({
 
+app.use('/graphql', express_graphql({
     schema: buildSchema(typeDefs),
     rootValue: root,
     graphiql: true
 }));
 
 app.route("/").get((request, response) => {
-	response.send("Rendered handlebars template here");
+	response.redirect("/api/user/login")
 });
 
-app.use("/node_modules", serveStatic(path.resolve(__dirname, "../node_modules")));
 app.use("/", serveStatic(path.resolve(__dirname, STATIC_ROOT)));
 
 app.listen(PORT, () => {
```

Comment:
should add a log message here (similar to the one removed) to the effect that 'the app was started'